src/Controller/Admin/AdminUsersController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Repository\UserRepository;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class AdminUsersController extends AbstractController
+{
+    private UserRepository $userRepository;
+
+    public function __construct(UserRepository $userRepository)
+    {
+        $this->userRepository = $userRepository;
+    }
+
+    /**
+     * @Route("/admin/users", name="app_admin_users")
```

Comment:
este o metoda de a pune un prefix pe ruta pentru toate controllerele dintr-un namespace. Poti sa incerci sa adaugi aici https://github.com/marceldobrica/training/blob/main/config/routes/annotations.yaml#L4 inca o sectiune care sa fie ceva gen: 
controllers_admin:
    resource: '../../src/Controller/Admin'
    type: annotation
    # this is added to the beginning of all imported route URLs
    prefix: '/admin'
    # this is added to the beginning of all imported route names
    name_prefix: 'admin_'
    
Daca ai altceva de lucru, da-i direct resolve la commentul asta, ca nu e o chestie importanta